### ISSUE TITLE
Cannot deploy (jaws dash) due to missing module 'bluebird'

### ISSUE THREAD

**Issue by DALDEI** [binoculars/awsm-s3tokenvendor#4]
Following the directions exacty I get this error

```
Error running browserify bundle                                                                                                                           
{ [Error: Cannot find module 'bluebird' from '/tmp/lAwsmS3tokenvendorVendtoken@1446035815903/aws_modules/awsm-s3tokenvendor/vendtoken']                   
  stream:                                                                                                                                                 
   { _readableState:                                                                                                                                      
      { highWaterMark: 16,                                                                                                                                
        buffer: [],                                                                                                                                       
        length: 0,                                                                                                                                        
        pipes: [Object],                                                                                                                                  
        pipesCount: 1,                                                                                                                                    
        flowing: true,                                                                                                                 
...

```

Looking in the source I see this reference

```
 * For example if awsm-images had a 'dependency' of bluebird defined in its package.json, bluebird could NOT be used
 * in this file.  This file will live in <project>/aws_modules/awsm-images/thumbnail/index.js and the bluebird mod will
 * be included in <project>/node_modules/awsm-images/node_modules/bluebird.
```

and lower

```
var Promise = require('bluebird');
```

So it seems I need to find 'bluebird' from somewhere and install it into hode_modules

I am not familiar with node at all -- any suggestions on how to go about that ?
I'm trying 'npm find bluebird' but its hanging right now -- If I do find it, whats the process for getting it and putting it in the right place ?

Thanks

**Comment by binoculars** [comment]
I'll take a look tonight and see if I can replicate the error.

**Comment by DALDEI** [comment]
I guessed a few times and installed bluebird individually as a node module 
Then jaws dash worked .. But ended up packing two copies of bluebird 
The top level node module and the  nested module with on binoculars

**Comment by binoculars** [comment]
I got the same thing. It seems to be a flaw in the jaws CLI that browserify doesn't find the nested dependencies. I'll see if there's a workaround or fix for this. Maybe a postinstall script is necessary.

**Comment by binoculars** [comment]
Ok I just tested it. Should be working just fine now. I kind of did a little hack by adding `Promise` as a global when requiring this module, but I don't see another solution.
